src/renderer/css/components/TabBar.scss

```diff
@@ -0,0 +1,202 @@
+#main {
+  .TabBar {
+    border: none;
+    max-height: 40px;
+    flex: 1 1 auto;
+    display: flex !important;
```

Comment:
Why is this important needed here? I can't see this being overwritten in another place. If its not being applied then probably there's a clash somewhere.

Reply:
Not sure, it's been around for a while. All of this CSS is old I just had to wrap it in a new file and a new tag.